### PR TITLE
Domain reorg software

### PR DESCRIPTION
**Pull request by iankirkman**

### CODE REVIEW

File: _bioinfcomputing/software_languages.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: Getting Started with Common Languages and Tools
+last_modified_at: 2019-04-24
+primary_reviewers: k8hertweck
+---
+
+## Training Resources 
+
+Fred Hutch offers training resources for each of the common languages and tools listed below. For course descriptions and session notifications, go to Hutch Learning [here](http://www.fredhutch.io/). New class offerings will also be posted in the Coop Newsletter. Email `thecoop` to join the mailing list.
```
Comment (k8hertweck) [inline _bioinfcomputing/software_languages.md]: I'd revise this section as follows:

Fred Hutch offers training resources for many of the common languages and tools listed below. Course descriptions can be found at [fredhutch.io](http://www.fredhutch.io/), and course registration is tracked in Hutch Learning (only accessible to Fred Hutch employees through CenterNet; search for "fredhutch.io" to see all relevant courses). New class offerings will also be posted in the Coop Newsletter. Email `thecoop@fredhutch.org` to join the mailing list.

File: _bioinfcomputing/software_overview.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+
+Frequently in scientific research, we must write code in order to perform our desired analysis. Whether we are writing a short script for our own manuscript, or making an app we intend to share with other researchers, incorporating standard programming practices will enhance the readability and reproducibility of our work. Below, we provide an overview of Fred Hutch coding standards as well as resource pages for commonly used programming languages in biosciences research.
+
+## [Getting Started with Common Languages](/bioinfcomputing/software_languages/)
+
+Fred Hutch has several resources available for folks that are seeking instruction in a new programming language, version control (managing and sharing code), or navigating linux/shell scripting. If software development in research is new to you, or you wish to pick up a new skill, we recommend you start with the [Common Languages](/bioinfcomputing/software_languages/) page.
+
+The remainder of this section will presume a basic skill level in at least one common language, as well as a basic understanding of [GitHub](/bioinfcomputing/software_managecode/).
```
Comment (k8hertweck) [inline _bioinfcomputing/software_overview.md]: I think this does a great job accomplishing the goal we discussed previously: managing expectations for both experienced users and newbies.

File: _bioinfcomputing/software_overview.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+```
+git clone https://github.com/FredHutch/wiki-code-examples.git
+```
+
+For more information on using Github, see the [Managing and Sharing Code](/bioinfcomputing/software_managecode/) page in this wiki.
+
+
+### Submitting Your Code
+When you have completed a coding project that you would like to submit to the examples wiki, there are a few steps in Github to navigate the review process. You will want to push your code in a clean branch to the repository, and then submit a pull request on `github.com` for your branch that will kick off a review and merge process.
```
Comment (k8hertweck) [inline _bioinfcomputing/software_overview.md]: I personally prefer instructions for submission to be kept within the repo itself. If we do want to keep this here, there should be a description to how to get added to the GitHub org so it's possible to create a branch within the original repo.

File: _bioinfcomputing/software_languages.md (hunk 1 above)
Comment (iankirkman) [reply _bioinfcomputing/software_languages.md]: Updated except for the email address, because the wiki convention is to hide the details.

**Comment by iankirkman** [comment]
@k8hertweck I've addressed all of your comments and I think we're ready to merge. I'm also curious if you've taken a look at the new templates repo? https://github.com/FredHutch/wiki-templates

**Pull request merged**